src/main/java/org/jenkinsci/plugins/GithubAuthenticationToken.java

```diff
@@ -302,7 +302,14 @@ public GithubOAuthUserDetails getUserDetails(String username) {
         if (user != null) {
             List<GrantedAuthority> groups = new ArrayList<GrantedAuthority>();
             try {
-                for (GHOrganization ghOrganization : user.getOrganizations()) {
+                GHPersonSet<GHOrganization> orgs;
+                if(this.userName.equals(username)) {
```

Comment:
If GitHub scopes don't include the ability to read teams, then all authorizations for a user are wiped.  Need to fall back to the default method of determining authorizations publicly.